### ISSUE TITLE
make usage more intuitive

### ISSUE THREAD

**Issue by mokkin** [stevenengland/FinCalcR#9]
Right like the hardware model, the usage should be more intuitive. Without reading the manual it is hard to discover how to enter values in the correct order and way.
Maybe it is is a good idea to have a look on [other tools](https://www.zinsen-berechnen.de/zinsrechner.php). There they clearly ask you which value are you looking for and which other ones are needed.
This is mainly a limit for the existing hardware calculator and not necessary for the software version.

**Comment by stevenengland** [comment]
You're completely right. Until verision 1.0 the overall goal is to adapt the mimic of the hardare based calculator so that existing documentation (manual, schooling book) can be reused. So it might also be desirable to port content from the manual or the schooling book to a wiki or similar so that everybody can use this version of the software calculator.

For upcoming versions after 1.0 it is the goal to
* create a more intuitive UI based on the existing one (like to give every function a specific button instead of sharing the buttons)
* better integrate help content (hints, popups, links to knowledge sites, ...)
* port the software to the web and maybe mobile apps (like with the help of Platform Uno)
